reliability-engineering/terraform/modules/concourse-monitoring/grafana.tf

```
@@ -57,11 +57,11 @@ data "template_file" "concourse_grafana_container_def" {
   template = "${file("${path.module}/files/grafana-container-def.json")}"
 
   vars {
-    deployment      = "${var.deployment}"
-    grafana_url     = "${local.grafana_url}"
-    database_host   = "${aws_db_instance.concourse_grafana_db.endpoint}"
-    aws_account_id  = "${data.aws_caller_identity.account.account_id}"
-    github_team_ids = "${join(",", var.grafana_allowed_github_team_ids)}"
+    deployment                   = "${var.deployment}"
+    grafana_url                  = "${local.grafana_url}"
+    database_host                = "${aws_db_instance.concourse_grafana_db.endpoint}"
+    aws_account_id               = "${data.aws_caller_identity.account.account_id}"
+    github_allowed_organizations = "${join(",", var.grafana_github_allowed_organizations)}"
```

```suggestion
    github_allowed_organizations = "${join(" ", var.grafana_github_allowed_organizations)}"
```

https://grafana.com/docs/grafana/latest/auth/github/#allowed-organizations

alleges that it is space delimited

---

https://github.com/grafana/grafana/blob/06bf7e8ef1af566e4c66a30a9e1651bf6552d1f1/pkg/login/social/social.go#L123
https://github.com/grafana/grafana/blob/06bf7e8ef1af566e4c66a30a9e1651bf6552d1f1/pkg/util/strings.go#L30
Either should work

---

yeah bit ambiguous ... we only have one org :see_no_evil: